lib/matobj2.gd

```diff
@@ -1222,6 +1231,9 @@ DeclareOperation( "CompanionMatrix",
 ##  of <Ref Oper="ShallowCopy"/>.
 ##  If <A>list</A> is a nested list then it is <E>not</E> guaranteed
 ##  that also the entries of <A>list</A> are copied.
+##  <P/>
+##  If the global option <C>Check</C> is set to <K>false</K> then
```

Comment:
I note that the majority of our existing global options are all-lowercase. I wonder if we have established any rules on this anywhere?